### DISCUSSION TITLE
v2.0 Development gained Momentum

### DISCUSSION THREAD

**Discussion by Dhruvacube** [Creatrix-Net/yondaime-hokage#206] [Announcements]
### Yup it is
As you could see the v2.0 development of bot got almost stopped since I had lot of works. But now with @hacktoberfest coming we have got a lot of contributors and got some quality PR's, which actually fixed some grave broken issue.

Thanks to them the v2.0 development of bot has gained momentum :) Yaay!!!!

Thanks contributors ❤️

**Comment by eitozx** [comment]
😳 That's great to know. Cheers! 🍻

**Reply by Dhruvacube** [reply]
Its a thanks to you @EitoZX , you fixed my broken code :D

**Reply by eitozx** [reply]
OO, aaye good to know that it helped you :> 
means a lot to me as well :D
